ec2utils/ec2uploadimg/lib/ec2utils/ec2uploadimg.py

```diff
@@ -268,6 +277,7 @@ def _create_block_device_map(self, snapshot):
     # ---------------------------------------------------------------------
     def _create_image_root_volume(self, source):
         """Create a root volume from the image"""
+        temporary_key_created = False
```

Comment:
?? why is this needed here?

Reply:
It's a leftover from the previous implementation. I forgot to remove it again.